mteb/tasks/Retrieval/XMarketESRetrieval.py

```diff
@@ -0,0 +1,58 @@
+import datasets
+
+from ...abstasks.AbsTaskRetrieval import AbsTaskRetrieval
+from ...abstasks.BeIRTask import BeIRTask
+
+
+class XMarketES(AbsTaskRetrieval):
+    _EVAL_SPLIT = 'test'
+    _LANG = 'es'
+
+    @property
+    def description(self):
+        return {
+            "name": "XMarketES",
+            "hf_hub_name": "jinaai/xmarket_ml",
```

Comment:
Why is it called `xmarket_ml`, not `xmarket_es` like the `xmarket_de` one?

Reply:
It's the multilingual (ml) version of the dataset. I'll actually just make this XMartket retrieval task multilingual and merge it with the XMarketDE task.